### ISSUE TITLE
Shower template in  need of improvement to support accessible presentations

### ISSUE THREAD

**Issue by swickr** [w3c/presentation-viewer#31]
Léonie Watson says that while the shower template has improved, there is still work to do.

We should be able to conduct the WCAG review ourselves, but adding @LJWatson should she wish to comment.
